doc/changes.md

```diff
@@ -1,5 +1,91 @@
 # Changelog
 
+## Release v0.9.0 (20YY/03/31)
+
+[![Digital Object Identifier for PyGMT v0.9.0](https://zenodo.org/badge/DOI/10.5281/zenodo.7772533.svg)](https://doi.org/10.5281/zenodo.7772533)
+
+### Highlights
+
+* 🎉 **Ninth minor release of PyGMT** 🎉
+* Add function to load raster tile maps using contextily ([#2125](https://github.com/GenericMappingTools/pygmt/pull/2125))
+* Add Figure.tilemap to plot XYZ tile maps ([#2394](https://github.com/GenericMappingTools/pygmt/pull/2394))
```

Comment:
Prefer to have `fig.tilemap` on top of `pygmt.load_tilemap` :slightly_smiling_face:

```suggestion
* Add Figure.tilemap to plot XYZ tile maps ([#2394](https://github.com/GenericMappingTools/pygmt/pull/2394))
* Add function to load raster tile maps using contextily ([#2125](https://github.com/GenericMappingTools/pygmt/pull/2125))
```